components/ActiveClinicData.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <v-container v-show="this.$store.getters.isUserLoggedIn">
```

Comment:
The linter can help us avoid using `this` in markup

Reply:
nice!